### ISSUE TITLE
Wrong resolution to Vue 2 when in Vue 3 project

### ISSUE THREAD

**Issue by hongquan** [Demivan/fluent-vue#740]
I'm using `fluent-vue` v3.0.0 in a Vue 3 project. When building, I got this error:

```
❯ yarn build                                        
yarn run v1.22.17
$ vue-tsc --noEmit && vite build
node_modules/fluent-vue/index.d.ts:19:40 - error TS2709: Cannot use namespace 'Vue' as a type.

19   interface ComponentOptions<V extends Vue> {
                                          ~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by Demivan** [comment]
Hi. Thanks for the report.

Can you try version 3.0.0-SNAPSHOT.1?
I'll publish it as 3.0.1 if everything is working for you.

**Comment by hongquan** [comment]
@Demivan Thank you for quick response. The `3.0.0-SNAPSHOT.2` fixes it.

**Comment by Demivan** [comment]
Fix is now deployed as a `3.0.1` version
